src/administrative-sdk/speech-challenge/speech-challenge-controller.js

```diff
@@ -54,7 +60,7 @@ export default class SpeechChallengeController {
 
     return this._connection._secureAjaxGet(url)
       .then(data => {
-        const challenge = new SpeechChallenge(data.id, data.topic, data.referenceAudioUrl);
+        const challenge = new SpeechChallenge(data.id, data.topic, data.referenceAudioUrl, data.srtUrl, data.imageUrl);
```

Comment:
If you would apply object destructoring you would only have to pass the `data` object to the SpeechChallenge constructor.